src/main/java/seedu/address/logic/commands/SetConsultationFeeCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.math.BigDecimal;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.record.Statistics;
+
+/**
+ * Sets the consultation fee of the clinic.
+ */
+public class SetConsultationFeeCommand extends Command {
+
+    public static final String COMMAND_WORD = "setconsultfee";
```

Comment:
Could make the command word shorter?

Reply:
I personally think the length is fine, as it won't be used often and it is already shortened. 
But I'll add a alias setfee, though it might overlap with any future potential additions (misc fees).